src/sipsess/listen.c

```diff
@@ -253,7 +253,7 @@ static void target_refresh_handler(struct sipsess_sock *sock,
 		return;
 	}
 
-	if (got_offer || is_invite) {
```

Comment:
We think that here a negative reply should be sent if the SDP is not allowed.

Reply:
Done.